app/controllers/api/databases.php

```diff
@@ -4202,6 +4202,245 @@ function updateAttribute(
         $response->dynamic($document, Response::MODEL_DOCUMENT);
     });
 
+App::put('/v1/databases/:databaseId/collections/:collectionId/documents/:documentId')
+    ->alias('/v1/database/collections/:collectionId/documents/:documentId')
```

Comment:
We don't need an alias for a new route, it's used for backwards compatibility if we change paths

```suggestion
```